plane/source/index.rst

```diff
@@ -105,23 +105,16 @@ Monitor https://discuss.ardupilot.org/c/arduplane for plane-related announcement
    :maxdepth: 4
    :hidden:
    
-    Introduction <docs/introduction>
+    Introduction <docs/plane-introduction>
     AutoPilot Hardware <docs/common-autopilots>
+    Ground Control Stations <docs/common-GCS>
     First Time Setup <docs/arduplane-setup>
     First Flight and Tuning <docs/first-flight-landing-page>
     QuadPlane Setup and Operation <docs/quadplane-support>
+    Mission Planning <docs/common-mission-planning>
```

Comment:
I noticed this is also under Ground Control Stations. Do you think it belongs better there or on the 1st level?

Reply:
both places....Randy wanted it on the left....and we may add to the GCS landing page